Demos/LightboxDemo/LightboxDemo/ViewController.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+import Lightbox
+
+class ViewController: UIViewController {
+
+  lazy var lightboxView: LightboxView = { [unowned self] in
```

Comment:
Just would like to ask, why do you need to use [unowned self] here, as I don't understand :smile: